docs/core/rules.rst

```diff
@@ -0,0 +1,187 @@
+
+* :ref:`forms`: It's a common use case for assistants to collect form-like
+  data from the user. Both, activation of forms as well as handling of unexpected
+  events as part of a form, are often following fixed paths.
+
+**Don't overuse rules**. They are great to handle small specific conversation parts but
+unlike :ref:`stories` rules don't have the power to generalize to unseen conversation
+paths. Combine rules and stories to make your assistant robust and able to handle
+real user behavior.
+
+Writing a Rule
+--------------
+
+Before you start writing rules, you have to make sure that the :ref:`rule-policy`
+is added to your model configuration:
+
+.. code-block:: yaml
+
+    policies:
+    - ... # Other policies
+    - name: RulePolicy
+
+Rules can then be added to the ``rules`` section of your training data.
+
+Rules for the Conversation Start
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To write a rule which only applies at the beginning of a conversation, start with the
+intent which starts the conversation and then add the actions which your assistant
+should perform in response to that.
+
+.. code-block:: yaml
+
+    rules:
+
+    - rule: Say `hello` when the user starts a conversation with intent `greet`
+      steps:
+      - intent: greet
+      - action: utter_greet
+
+.. _rules-applying-anytime:
+
+Rules which Apply Anytime
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To indicate that a rule can apply at any point in a conversation, specify ``...`` as
+first conversation step. The ``...`` indicate that the rule applies independent of the
+previous conversation.
+
+.. code-block:: yaml
+
+    rules:
+
+    - rule: Say `hello` whenever the user sends a message with intent `greet`
+      steps:
+      - ...
+      - intent: greet
+      - action: utter_greet
+
+This example rule applies at the start of conversation as well as when the user decides
+to a send a message with an intent ``greet`` in the middle of an ongoing conversation.
+
+Rules with Pre-Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Rules can describe requirements which have to be fulfilled for the rule to be
+applicable. To do so, add any information about the prior conversation, before the
+``...``:
+
+.. code-block:: yaml
+
+    rules:
+
+    - rule: Only say `hello` when the user provided a name
+      steps:
+      - slot: user_provided_name
+        value: true
+      - ...
+      - intent: greet
+      - action: utter_greet
+
+Not Predicting `action_listen` at the End of a Rule
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Rules default to wait for the next user message when they finished their last step:
+
+.. code-block:: yaml
+
+    rules:
+
+    - rule: Rule which will wait for user message when it was applied
+      steps:
+      - intent: greet
+      - action: utter_greet
+      # Every rule implicitly includes a prediction for `action_listen` as last step.
+      # This means that Rasa Open Source will wait for the next user message.
+      - action_listen
```

Comment:
this looks weird in the docs. I don't think unexperienced user will get it

Reply:
This will also change with the changes from the feedback